packages/amplication-client/src/VersionControl/PendingChange.tsx

```diff
@@ -80,6 +80,13 @@ export const changeOriginMap = {
         type: "Service Settings",
         pluralTypeName: "Service Settings",
       },
+      [models.EnumBlockType.Package]: {
+        relativeUrl: `settings/packages`,
+        icon: "settings",
+        displayName: "Service packages",
```

Comment:
```suggestion
        displayName: change.displayName,
```

Reply:
@mulygottlieb will handle in the next PR